### PR TITLE
[l2-load-balancer] Documentation fixes + declaring module as experimental

### PR DESCRIPTION
**Pull request by apolovov**
## Description
Minor documentation fixes and declaration as experimental.

## Why do we need it in the patch release (if we do)?
We figured out that the design is bad and it is better not to apply the module in user's environments.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: l2-load-balancer
type: fix
summary: Module is declared as experimental.
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/stage'...

File: ee/modules/381-l2-load-balancer/crds/doc-ru-l2-load-balancer.yaml [hunk 1]
```diff
@@ -3,7 +3,10 @@ spec:
   - name: v1alpha1
     schema: &schema
       openAPIV3Schema:
-        description: Интерфейс для настройки распределённого балансировщика в режиме L2.
+        description: |
+          Интерфейс для настройки распределённого балансировщика в режиме L2.
+
+          **Важно!** Модуль **экспериментальный**, не используйте его в production-окружении.
```
Comment (konstantin-axenov) [inline ee/modules/381-l2-load-balancer/crds/doc-ru-l2-load-balancer.yaml:9]:
```suggestion
          **Важно!** Модуль **экспериментальный** и находится в процессе активного развития. Функциональность и API могут существенно измениться.

File: ee/modules/381-l2-load-balancer/crds/l2-load-balancer.yaml [hunk 2]
```diff
@@ -21,8 +21,12 @@ spec:
     schema: &schema
       openAPIV3Schema:
         type: object
-        description: The interface for configuring distributed L2 load balancers.
+        description: |
+          The interface for configuring distributed L2 load balancers.
+
+          **Caution!** The module is **experimental**, don't use it in production environment.
```
Comment (konstantin-axenov) [inline ee/modules/381-l2-load-balancer/crds/l2-load-balancer.yaml:27]:
```suggestion
          **Caution!** The module is **experimental**. It is actively developed and its features and API might significantly change in the future.
```

File: ee/modules/381-l2-load-balancer/docs/README.md [hunk 3]
```diff
@@ -2,13 +2,17 @@
 title: "The l2-load-balancer module"
 ---
 
-The module implements an improved (relative to the standard [[L2 mode in MetalLB]](../../modules/380-metallb/#layer-2-mode)) balancing mechanism for services in bare metal clusters when there is no option to use cloud load balancers or [MetalLB](../../modules/380-metallb/#mode-bgp) in BGP mode with Equal-cost multi-path (ECMP) configured.
+{% alert level="warning" %}
+The module is **experimental**, don't use it in production environments.
```
Comment (konstantin-axenov) [inline ee/modules/381-l2-load-balancer/docs/README.md:6]:
```suggestion
The module is **experimental**. It is actively developed and its features and API might significantly change in the future.
```

File: ee/modules/381-l2-load-balancer/docs/README_RU.md [hunk 4]
```diff
@@ -2,13 +2,17 @@
 title: "Модуль l2-load-balancer"
 ---
 
-Модуль реализует улучшенный (относительно стандартного [[режима L2 в MetalLB]](../../modules/380-metallb/#режим-layer-2)) механизм балансировки для сервисов в кластерах bare metal, где нет возможности воспользоваться облачным балансировщиком или [MetalLB](../../modules/380-metallb/#режим-bgp) в режиме BGP с настроенным Equal-cost multi-path (ECMP).
+{% alert level="warning" %}
+Модуль **экспериментальный**, не используйте его в production-окружениях.
```
Comment (konstantin-axenov) [inline ee/modules/381-l2-load-balancer/docs/README_RU.md:6]:
```suggestion
Модуль **экспериментальный** и находится в процессе активного развития. Функциональность и API могут существенно измениться.
```

File: ee/modules/381-l2-load-balancer/docs/README_RU.md [hunk 5]
```diff
@@ -2,13 +2,17 @@
...
 
 Основан на решении [MetalLB](https://metallb.universe.tf/).
 
 ## Принцип работы в сравнении с режимом L2 модуля MetalLB
 
-[[MetalLB в режиме L2]](../../modules/380-metallb/#режим-layer-2) позволяет заказать _Service_ с типом `LoadBalancer`, работа которого основана на том, что в пиринговой сети узлы-балансировщики имитируют ARP-ответы от "публичного" IP. Данный режим имеет существенное ограничение — единовременно лишь один балансировочный узел обрабатывает весь входящий трафик данного сервиса. Соответственно:
+[MetalLB в режиме L2](../../modules/380-metallb/#режим-layer-2) позволяет заказать _Service_ с типом `LoadBalancer`, работа которого основана на том, что в пиринговой сети узлы-балансировщики имитируют ARP-ответы от "публичного" IP. Данный режим имеет существенное ограничение — единовременно лишь один балансировочный узел обрабатывает весь входящий трафик данного сервиса. Соответственно:
 
 * Узел, выбранный в качестве лидера для "публичного" IP становится "узким местом" без возможности горизонтального масштабирования.
```
Comment (Zhbert) [inline ee/modules/381-l2-load-balancer/docs/README_RU.md:17]:
```suggestion
* Узел, выбранный в качестве лидера для "публичного" IP-адреса, становится "узким местом" без возможности горизонтального масштабирования.
```

File: ee/modules/381-l2-load-balancer/docs/README.md [hunk 6]
```diff
@@ -2,13 +2,17 @@
...
 
 Based on [MetalLB](https://metallb.universe.tf/) solution.
 
 ## Principle of operation compared to L2 mode in MetalLB module
 
-[[MetalLB in L2 mode]](../../modules/380-metallb/#layer-2-mode) allows to order _Service_ with `LoadBalancer` type, the operation of which is based on the fact that balancing nodes simulate ARP-responses from the "public" IP in a peering network. This mode has a significant limitation — only one balancing node handles all the incoming traffic of this service at a time. Therefore:
+[MetalLB in L2 mode](../../modules/380-metallb/#layer-2-mode) allows to order _Service_ with `LoadBalancer` type, the operation of which is based on the fact that balancing nodes simulate ARP-responses from the "public" IP in a peering network. This mode has a significant limitation — only one balancing node handles all the incoming traffic of this service at a time. Therefore:
```
Comment (kirkonru) [inline ee/modules/381-l2-load-balancer/docs/README.md:15]:
```suggestion
[MetalLB in L2 mode](../../modules/380-metallb/#layer-2-mode) allows for provision a _Service_ of the `LoadBalancer` type, the operation of which is based on the fact that balancing nodes simulate ARP responses from the "public" IP in a peering network. This mode has a significant limitation — only one balancing node handles all the incoming traffic of this service at a time. Therefore:
```

**Review by kirkonru — APPROVED** [review]

**Review by Zhbert — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [8266](https://github.com/deckhouse/deckhouse/pull/8266) to the branch [release-1.59](https://github.com/deckhouse/deckhouse/tree/release-1.59) successful!

**Pull request merged**